test/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "bedrock-vc-verifier-test",
+  "version": "0.0.1-0",
+  "description": "Bedrock VC Verifier Test",
+  "main": "test",
+  "private": true,
+  "scripts": {
+    "test": "node --preserve-symlinks test.js test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/bedrock-key-http"
+  },
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "http://digitalbazaar.com"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-key-http/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-key-http",
```

Comment:
trying out multi-line comments.  some of this stuff is not accurate (bedrock-key), but we don't need this stuff in the test package file, this file should have bare minimum required.

Reply:
I cut it down and removed `bedrock-key` here: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/1/commits/649b8d87e76e10658c6c700878807b883d8587e1